utilsForFrontend.mjs

```diff
@@ -464,20 +464,20 @@ export function sigmoid(x) {
 export function isVisible(fnodeOrElement) {
     // This could be 5x more efficient if https://github.com/w3c/csswg-drafts/issues/4122 happens.
     const element = toDomElement(fnodeOrElement);
-    // Avoid reading ``display: none`` due to Bug 1381071
     const elementRect = element.getBoundingClientRect();
-    if (elementRect.width === 0 && elementRect.height === 0) {
+    const elementStyle = getComputedStyle(element);
+    // Alternative to reading ``display: none`` due to Bug 1381071.
+    if (elementRect.width === 0 && elementRect.height === 0 && elementStyle.overflow !== 'hidden') {
```

Comment:
Shouldn't this be `elementStyle.overflow === 'hidden'`?  As you say below, "Zero-sized ancestors don’t make descendants hidden unless the descendant has overflow: hidden".

Reply:
The check you're thinking of is done inside the `for` loop below. This first check is a proxy for checking `display: none`, for which just checking `width` and `height` === 0 is not sufficient as I learned while working on #122 .

Reply:
Ah, so this is a check strictly for the visibility of the element passed in, not any of its descendents. A child, for example, could still be visible. Callers will have to be aware of this. That'd be worth documenting in the doclet, as I wasn't front-end-savvy enough to realize it.

Reply:
Hm... I'm not sure I follow what you want me to document. In general, this method is to check if the element passed in is visible. It doesn't make any claims about that element's descendants.

Reply:
We discussed this synchronously, and we're just going to leave off any additional comments for now.